index.js

```diff
@@ -13,6 +13,7 @@ const MAX_TAG_VALUE_LENGTH = 256;
 const SANITIZATION_CHARACTER = '_';
 const ROLE_SESSION_NAME = 'GitHubActions';
 const REGION_REGEX = /^[a-z0-9-]+$/g;
+const AWS_CURRENT_REGION = process.env.AWS_REGION
```

Comment:
We shouldn't set this as a global variable - instead use `process.env` where the value is needed.